docs/content/direct/release-notes.md

```diff
@@ -4,7 +4,8 @@ The following sections list the known issues for each release. The issue list is
 
 ## 0.25.0 and its candidates
 
-The main advance in this release is finishing the implementation of the create-only feature. It is now available for use.
+* The main advance in this release is finishing the implementation of the create-only feature. It is now available for use.
+* The transport controller's `max-num-wrapped` flag is defaulted to 1 in the core Helm chart, as a work around of bug https://github.com/kubestellar/kubestellar/issues/2511.
```

Comment:
Changing the default is good, but does not work around the bug --- any user can still set a value that will get handled wrongly. The fact that values leading to multiple ManifestWork object with multiple workload objects in them are not handled properly should be noted in the "remaining limitations" section --- in all the releases that have this problem.